lib/ransack/helpers/form_helper.rb

```diff
@@ -94,7 +94,7 @@ def initialize(search, attribute, args, params)
           @current_dir    = existing_sort_direction
           @label_text     = extract_label_and_mutate_args!(args)
           @options        = extract_options_and_mutate_args!(args)
-          @hide_indicator = @options.delete :hide_indicator
+          @hide_indicator = @options.delete :hide_indicator || Ransack.options[:hide_sort_order_indicators]
```

Comment:
`:hide_indicator` needs to be wrapped in parentheses here, otherwise it is evaluated by Ruby as `@options.delete (:hide_indicator || Ransack.options[:hide_sort_order_indicators])` which isn't what we want. It needs to be:

```ruby
          @hide_indicator = @options.delete(:hide_indicator) ||
            Ransack.options[:hide_sort_order_indicators]
```